static/usr/lib/systemd/system-generators/rtc-sys-time-init-generator

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# This generator enables rtc-sys-time-init.service by hooking it into
+# systemd-timesyncd.service with symlink in $1/systemd-timesyncd.service.wants 
+# when this is requested by kernel command line flag 'rtc_sys_time_init'.
+
+DEBUG_ENABLE="false"
+SERVICE="rtc-sys-time-init.service"
+SERVICE_PATH="/usr/lib/systemd/system/$SERVICE"
+KERNEL_CMDLINE_ENABLE_FLAG="rtc_sys_time_init"
+HOOK_TO_UNIT="sysinit.target"
+
+log() {
+    echo "$@" >> "/dev/kmsg"
+}
+
+debug() {
+    if [ "$DEBUG_ENABLE" = "true" ]; then
+        log "$@"
+    fi
+}
+
+read_kernel_command_line() {
```

Comment:
There is `get-arg` that I have added in another PR that you can use.

Reply:
`/usr/libexec/core/get-arg`

Reply:
Thanks, back-ported this into my code to enable testing.